gui/system/forms.py

```diff
@@ -2491,6 +2491,11 @@ def clean_cert_certificate(self):
 
         if nmatches > 1:
             self.instance.cert_chain = True
+
+        try:
+            cert_info = load_certificate(self.instance.cert_certificate)
+        except:
```

Comment:
"except:" is usually a bad thing, lets try to avoid that.

What is the exception it raises when that happens? Lets use that.

Reply:
Besides that, note this introduces a new flake error. `cert_info` is not used.

Reply:
except crypto.Error: